.github/workflows/build_and_test.yml

```diff
@@ -11,11 +11,10 @@ on:
       - synchronize
 
 jobs:
-  linux:
-    name: Linux
+  browser:
+    name: Browser
```

Comment:
I've changed this such that the browser core build and test runs first, then only when that succeeds do we run the rest of the builds. This will save somewhat on resources. We'll also need to set this check as required in the branch protection once this is merged.

Reply:
This is more like web app test. Perhaps call it  "Web Tests".